casper/src/test/scala/coop/rchain/casper/MultiParentCasperTestUtil.scala

```diff
@@ -100,11 +100,12 @@ object MultiParentCasperTestUtil {
     implicit val log: Log.NOPLog[Task]     = new Log.NOPLog[Task]
     implicit val metricsEff: Metrics[Task] = new metrics.Metrics.MetricsNOP[Task]
     (for {
+      //TODO consider reverting back to the production-used variant
       activeRuntime <- Runtime
                         .createWithEmptyCost[Task, Task.Par](
-                          storageDirectory,
-                          storageSize,
-                          StoreType.LMDB
+                          Paths.get("/not/a/path"),
+                          -1,
+                          StoreType.InMem
```

Comment:
Is InMem this something like a Java HashTable?  If so, I think it is risky to use that.  I'd rather try to disable `fsync` in LMDB.

Reply:
See `metasync` and `sync` options at https://lmdb.readthedocs.io/en/release/#environment-class

Reply:
WDYM by risky? Do you expect more errors? Or more false-positives?

If the latter, I'd say having 200 tests not targeted at the false-positives exercise the tupplespace is an overkill and a waste.

Re concurrency semantics - @dzajkowski WDYT?

Reply:
> WDYM by risky? Do you expect more errors? Or more false-positives?

I mean any difference in behavior compared to actual LMDB.  Take for instance the long standing bug of segfaulting at shutdown.  Will it happen with InMem?  I think it should and I bet it won't.